app/views/application/_breadcrumbs.html.erb

```diff
@@ -1,4 +1,7 @@
+<% inverse ||= false %>
```

Comment:
With a caveat that I'm missing a ton of context coming into this fairly cold, but I'm not clear why you've set this default, and it also doesn't seem to be relevant to the intent of the commit of creating new partials - could you split this into a separate commit with more explanation around the change?